src/latexify/codegen/function_codegen_test.py

```diff
@@ -862,6 +862,34 @@ def test_use_set_symbols_compare(code: str, latex: str) -> None:
     assert function_codegen.FunctionCodegen(use_set_symbols=True).visit(tree) == latex
 
 
+def test_generate_numpy_zeros():
+    tree = ast.parse(
+        textwrap.dedent(
+            """
+        def f(a, b):
+            return np.zeros((a,b))
+            """
+        )
+    ).body[0]
+    latex = "f(a, b) = 0^{{a} \\times {b}}"
+    assert isinstance(tree, ast.FunctionDef)
```

Comment:
We don't need complete parsing. It is enough to obtain only `ast.Call`.
```suggestion
    tree = ast_utils.parse_expr("zeros((a, b))")
    assert isinstance(tree, ast.Call)
```